fbw-a32nx/src/wasm/systems/a320_systems/src/hydraulic/sfcc/flaps_channel.rs

```diff
@@ -252,7 +277,119 @@ impl FlapsChannel {
+                }
+                (Some(cas1), _)
+                    if cas1 > self.kts_100
+                        && cas1 < self.kts_210
+                        && SlatFlapControlComputerMisc::in_enlarged_target_range(
+                            self.flaps_feedback_angle,
+                            self.conf1_flaps,
+                        ) =>
+                {
+                    self.flap_auto_command_angle = self.conf1_flaps
+                }
+                (_, Some(cas2))
+                    if cas2 > self.kts_100
+                        && cas2 < self.kts_210
+                        && SlatFlapControlComputerMisc::in_enlarged_target_range(
+                            self.flaps_feedback_angle,
+                            self.conf1_flaps,
+                        ) =>
+                {
+                    self.flap_auto_command_angle = self.conf1_flaps
+                }
+                (Some(cas1), Some(cas2))
+                    if ((cas1 <= self.kts_100 && cas2 >= self.kts_210)
+                        || (cas1 >= self.kts_210 && cas2 <= self.kts_100))
+                        && SlatFlapControlComputerMisc::between_0_1f_enlarged_target_range(
+                            self.flaps_feedback_angle,
+                        ) =>
+                {
+                    // TODO: implement startup inhibition
+                    self.flap_auto_command_angle = self.flaps_feedback_angle
+                }
+                (Some(cas1), _)
+                    if cas1 > self.kts_100
+                        && cas1 < self.kts_210
+                        && SlatFlapControlComputerMisc::between_0_1f_enlarged_target_range(
+                            self.flaps_feedback_angle,
+                        ) =>
+                {
+                    // TODO: implement startup inhibition
+                    self.flap_auto_command_angle = self.flaps_feedback_angle
+                }
+                (_, Some(cas2))
+                    if cas2 > self.kts_100
+                        && cas2 < self.kts_210
+                        && SlatFlapControlComputerMisc::between_0_1f_enlarged_target_range(
+                            self.flaps_feedback_angle,
+                        ) =>
+                {
+                    // TODO: implement startup inhibition
+                    self.flap_auto_command_angle = self.flaps_feedback_angle
+                }
+                (None, None) => self.flap_auto_command_angle = self.conf1f_flaps,
+                (_, _) => self.flap_auto_command_angle = self.flaps_feedback_angle,
+            }
+            self.flap_auto_command_active = true;
+        }
+    }
+
     fn generate_flap_angle(&mut self, adirs: &impl AdirsMeasurementOutputs) -> Angle {
+        if !self.is_powered && self.power_duration > Self::TRANSPARENCY_TIME {
+            self.flap_auto_command_active = false;
+            self.flap_auto_command_engaged = false;
+            self.flap_auto_command_angle = Angle::ZERO;
```

Comment:
Maybe we can put such resets into a separate function which is called when power is lost.